modules/ting_das/ting_das.module

```diff
@@ -0,0 +1,378 @@
+<?php
+
+/**
+ * @file
+ * Handle the integration of digital article service (DAS) into the CMS.
```

Comment:
Do we have a reference to the documentation for this service? A link or so would be nice.